spiceaidocs/docs/data-connectors/databricks.md

```diff
@@ -8,40 +8,35 @@ pagination_prev: null
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-Databricks as a connector for federated SQL query against Databrick's [Delta Lake](https://docs.databricks.com/en/delta/index.html). 
+Databricks as a connector for federated SQL query against Databricks using [Spark Connect](https://www.databricks.com/blog/2022/07/07/introducing-spark-connect-the-power-of-apache-spark-everywhere.html). 
 
 ## Configuration
 
-`spice login databricks` can be used to configure secrets for the Spice runtime (including AWS object store keys). 
+`spice login databricks` can be used to configure the Databricks access token for the Spice runtime. 
 
 ### Parameters
-- `endpoint`: The HTTPS endpoint of the Databricks host storing the desired tables.
-- `timeout`: The timeout duration for calls to underlying object store in string format. Default to `300s`.
+- `endpoint`: The endpoint of the Databricks instance.
+- `databricks-cluster-id`: The ID of the compute cluster in Databricks to use for the query.
 
 ### Auth
 
-An active personal access token for the Databricks instance is required (equivalent to `DATABRICKS_TOKEN`). 
-Other keys provided in the secret are directly passed to the underlying secret store (e.g. `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` if backed by AWS S3).
+An active personal access token for the Databricks instance is required (equivalent to `DATABRICKS_TOKEN`).
 
-By default Databricks connector will look for a secret named `databricks` with keys `token` and `AWS_DEFAULT_REGION`, 
-`AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`.
+By default the Databricks connector will look for a secret named `databricks` with keys `token`.
 
 Check [Secrets Stores](/secret-stores) for more details.
 
 <Tabs>
   <TabItem value="local" label="Local" default>
     ```bash
-    spice login databricks --token <access-token> --aws-region <aws-region> --aws-access-key-id <aws-access-key-id> --aws-secret-access-key <aws-secret-access-key>
+    spice login databricks --token <access-token>
     ```
 
     Learn more about [File Secret Store](/secret-stores/file).
   </TabItem>
   <TabItem value="env" label="Env">
     ```bash
     SPICE_SECRET_DATABRICKS_TOKEN=<access-token> \
-    SPICE_SECRET_DATABRICKS_AWS_DEFAULT_REGION=<aws-region> \
```

Comment:
We'll need to keep these for the Delta Lake Direct support.

Reply:
We'll update the docs after current release.